swiss-army-knife/admin-tools.yaml

```diff
@@ -119,38 +64,54 @@ spec:
         name: swiss-army-knife
     spec:
       tolerations:
-      - operator: Exists
+        - operator: Exists
       containers:
-      - name: swiss-army-knife
-        image: supporttools/swiss-army-knife
-        imagePullPolicy: IfNotPresent
-        securityContext:
-          privileged: true        
-        resources:
-          limits:
-            cpu: 1000m
-            memory: 1000Mi
-          requests:
-            cpu: 100m
-            memory: 100Mi
-        env:
-        - name: POD_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: POD_IP
-          valueFrom:
-            fieldRef:
-              fieldPath: status.podIP
-        - name: NODE_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.nodeName            
-        volumeMounts:
-        - name: rootfs
-          mountPath: /rootfs
-      serviceAccountName: swiss-army-knife         
+        - name: swiss-army-knife
+          image: supporttools/swiss-army-knife
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          resources:
+            limits:
+              cpu: 1000m
+              memory: 1000Mi
+            requests:
+              cpu: 100m
+              memory: 100Mi
```

Comment:
I know this was in place before, but just a thought, especially as it's a daemonset, shall we avoid subtracting resources from the node with requests?

Over-use of [limits is best avoided](https://home.robusta.dev/blog/stop-using-cpu-limits) as this can increase context switching/throttling.